live-examples/js-examples/string/string-slice.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">var str = 'The quick red fox jumped over the lazy dog\'s back.';
+console.log(str.slice(30)) // the lazy dog's back.
```

Comment:
Please use the standard `expected output:` format, and a blank line between each example, like:
```
console.log(str.slice(30));
// expected output: "the lazy dog's back."

console.log(str.slice(4,17));
// expected output: ...
```

Reply:
Also this line is missing the semicolon.

Reply:
Fixed the missing semicolon and reformatted the comments.